### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/nebula-python/commit/bbfd70e35c55b40b6805c77d8f4930ff09029d46#commitcomment-59918145

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/nebula-python/commit/6a78a557e5e8f49bf67122a144e9796c8d60c9d9

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: nebula2/common/ttypes.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ class ErrorCode:
   E_BALANCED = -2024
   E_NO_RUNNING_BALANCE_PLAN = -2025
   E_NO_VALID_HOST = -2026
-  E_CORRUPTTED_BALANCE_PLAN = -2027
+  E_CORRUPTED_BALANCE_PLAN = -2027
```
Comment (jsoref) [inline nebula2/common/ttypes.py:120]: Note that this appears to relate to a change in vesoft-inc/nebula#3297 which definitely hints that it's an API and thus the typo should probably be retained.

File: tests/test_data_type.py [hunk 2]
```diff
@@ -534,42 +534,42 @@ def test_path_api(self):
 
 class TestDatesetWrapper(TestBaseCase):
     def test_all(self):
-        data_set_warpper1 = DataSetWrapper(self.get_data_set())
-        data_set_warpper2 = DataSetWrapper(self.get_data_set())
+        data_set_wrapper1 = DataSetWrapper(self.get_data_set())
+        data_set_wrapper2 = DataSetWrapper(self.get_data_set())
```
Comment (jsoref) [inline tests/test_data_type.py:538]: Sometimes it's hard to tell, especially in tests, but, probably?

File: nebula2/common/ttypes.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ class ErrorCode:
   E_BALANCED = -2024
   E_NO_RUNNING_BALANCE_PLAN = -2025
   E_NO_VALID_HOST = -2026
-  E_CORRUPTTED_BALANCE_PLAN = -2027
```
Comment (Shylock-Hg) [inline nebula2/common/ttypes.py:120]: Please ignore this auto generated files.

**Comment by Shylock-Hg** [comment]
Why don't add it to github workflow?

**Comment by jsoref** [comment]
If you're interested in the workflow, I'm happy to. But typically I wait for an invitation.

**Comment by Shylock-Hg** [comment]
> If you're interested in the workflow, I'm happy to. But typically I wait for an invitation.

I think it's ok, how about you? @Aiee

File: tests/test_data_type.py (hunk 2 above)
Comment (Aiee) [reply tests/test_data_type.py:538]: Nice catch.

**Comment by Aiee** [comment]
> If you're interested in the workflow, I'm happy to. But typically I wait for an invitation.

Hello @jsoref, this tool indeed makes life easier. Could you please submit a PR to add the action? When you add the action, please ignore `/common /fbthrift /graph /meta /storage` under the `nebula2` directory since the files are auto-generated.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ Compatible with the v2.5.0 version of nebula-graph
 	- add TimeWrapper/DateTimeWrapper type to get timezone obtained from the server to calculate the local time
 	- Session supports reconnecting to different graph services
 - bugfix
-	- fix the interface valuesof Relationship and modify the interface propertysto propertieshttps://github.com/vesoft-inc/nebula-python/pull/113
+	- fix the interface values of Relationship and modify the interface properties to properties https://github.com/vesoft-inc/nebula-python/pull/113
```
Comment (jsoref) [inline CHANGELOG.md:8]: @greyli explained in https://github.com/vesoft-inc/nebula-python/pull/173#issuecomment-966016705 that this changelog entry was trying to say it was changing `propertys` to `properties`.

Would this be acceptable? ([[sic]](https://www.merriam-webster.com/dictionary/sic) is used to indicate it reproduces something as originally written (as opposed to how it should be written))
```suggestion
	- fix the interface values of Relationship and modify the interface `propertys` [sic] to `properties` https://github.com/vesoft-inc/nebula-python/pull/113
```

Comment (Aiee) [reply CHANGELOG.md:8]: Totally!

**Review by Aiee — APPROVED** [review]

**Pull request merged**